lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -41,10 +41,11 @@
             EmailFieldView: FieldViews.TextFieldView.extend({
                 fieldTemplate: field_text_account_template,
                 successMessage: function () {
-                    return this.indicators.success + StringUtils.interpolate(
+                    return HtmlUtils.joinHtml(
+                        this.indicators.success, StringUtils.interpolate(
```

Comment:
`this.indicators.success` does contain HTML. It didn't give any errors in the safe template run, but once I started to change other things around, this didn't work as is.